### DISCUSSION TITLE
Move the daily statistics from the left bar under the menu and unify it with Daily chart

### DISCUSSION THREAD

**Discussion by jmtmp** [Nachtzuster/BirdNET-Pi#182] [Ideas]
What about to drop left bar with statistic on page "Overview" and move it below menu, similarly to page "Daily chart". Unification would be also appreciated. Personally, I would prefer such info "X species today, XX in total; N detection today, NN on daily average".

**Comment by Svardsten53** [comment]
I agree and suggested this years ago in the McGuire version of the program.

**Reply by alexbelgium** [reply]
I've created a PR here : https://github.com/Nachtzuster/BirdNET-Pi/pull/191
